src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/LogConfiguration.java

```diff
@@ -25,13 +26,16 @@
     @Autowired
     private Environment env;
 
-    private void configureLog4j() throws IOException
+    private void configureLog4j()
     {
         for (String profile : env.getActiveProfiles()) {
             ClassPathResource resource = new ClassPathResource(profile + "/log4j.properties");
-            Properties properties = new Properties();
-            properties.load(resource.getInputStream());
-            PropertyConfigurator.configure(properties);
+
+            try (InputStream configFile = resource.getInputStream()) {
+                PropertyConfigurator.configure(configFile);
+                return;
+            } catch (IOException ignore) {
+            }
```

Comment:
if we want to log this we have to use a default logger ...this is the logging configuration @peterpunch